app/lib/time_limit_config.rb

```diff
@@ -62,7 +50,6 @@ def self.rules
     }
   end
 
-  def self.limits_for(rule)
-    rules[rule.to_sym]
-  end
+  private_constant 'Rule'
```

Comment:
You can avoid these with `class << self`.

Reply:
I know how the `private_class_method` can be avoided, but not the `private_constant`.